src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/updatelistingrequirements/UpdateListingRequirementsAboutToStartHandler.java

```diff
@@ -59,7 +59,6 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
                 schedulingAndListingFields.setOverrideFields(overrideFields);
             }
 
-            utils.generateInterpreterLanguageFields(overrideFields);

```

Comment:
Why have you removed this? It is still needed.
You should do something like
```suggestion
   if (isNull(overrideFields.getAppellantInterpreter())
            || isNull(overrideFields.getAppellantInterpreter().getInterpreterLanguage())) {
            overrideFields.setAppellantInterpreter(HearingInterpreter.builder()
                .interpreterLanguage(new DynamicList(null, null))
                .build());
    }
    languageList = utils.generateInterpreterLanguageFields(overrideFields.getAppellantInterpreter().getInterpreterLanguage());
    overrideFields.getAppellantInterpreter().setInterpreterLanguage(languageList);

```